Explorer/Assets/DCL/CommunicationData/URLHelpers/URN.cs

```diff
@@ -137,13 +133,32 @@ public URN Shorten()
                     if (index == -1) break;
                 }
 
-                return index != -1 ? originalUrn[..index] : originalUrn;
+                return index != -1 ? GetShortenedUrn(index) : this;
             }
 
             // TokenId is always placed in the last part for regular nfts
             index = originalUrn.LastIndexOf(':');
```

Comment:
Maybe this can go into an else and in that case you wont need to have    `   return index != -1 ? GetShortenedUrn(index) : this;`  twice, you can just call it after the else, wdyt?